scripts/prune_workers.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+set -e
+
+# Log in to Concourse
+
+fly -t networking-extensions login -c https://concourse.arp.cloudfoundry.org/
```

Comment:
fly -t networking-extensions login -c https://concourse.arp.cloudfoundry.org/ -u admin_concourse -p `grep CONCOURSE_ADMIN_PWD /concourse/.concourse.env | cut -d = -f 2`

Reply:
Note that this expects fly to be present, but we don't install it yet. I would add fly install into start.sh script, just after docker compose-up (and before cgroups setup):

```
# Install local fly
curl -o /usr/local/bin/fly -k 'https://localhost/api/v1/cli?arch=amd64&platform=linux'
chmod +x /usr/local/bin/fly
```